storage/retry_conformance_test.go

```diff
@@ -211,12 +210,24 @@ var methods = map[string][]retryFunc{
 			if err != nil {
 				return err
 			}
-			wr, err := io.Copy(ioutil.Discard, r)
+			wr, err := r.WriteTo(io.Discard)
 			if got, want := wr, len(randomBytesToWrite); got != int64(want) {
 				return fmt.Errorf("body length mismatch\ngot:\n%v\n\nwant:\n%v", got, want)
 			}
 			return err
 		},
+		func(ctx context.Context, c *Client, fs *resources, _ bool) error {
```

Comment:
Suggest mentioning the difference between this and the previous func and why we test both.